### PR TITLE
Add a GitHub pull request template

### PR DESCRIPTION
**Pull request by RussKie**
That's how it will look when a PR is opened:
image

### CODE REVIEW

**Comment by merriemcgaw** [comment]
Let's add a section for Accessibility testing - if they have or alter UI in their PR we should insist that they show us that Accessibility Insights Fast Pass run is clean, and that they add to the sample apps the UI for future accessibility testing. @Olina-Zhang can point to where the accessibility sample apps are. The accessibility insights tool is at [https://accessibilityinsights.io/](https://accessibilityinsights.io/)

**Comment by RussKie** [comment]
@merriemcgaw - updated

@Olina-Zhang - please let me know where the samples are. And anything else you feel is important to add.
Thank you

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- 
+- 
+
+## Accessibility testing  <!-- Remove this section if PR does not change UI -->
+
+<!-- 
+     If you added/modified UI you will need to show us that 
+     Accessibility Insights Fast Pass run is clean
+     
+     You can get Accessibility Insights tool from https://accessibilityinsights.io/
```
Comment (zsd4yr) [inline .github/PULL_REQUEST_TEMPLATE.md]: we should probably link to a wiki page that contains this info / a guide on how to use the tool

Comment (merriemcgaw) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes! I believe there are docs here, but it's critical that people know the what/why and the rules. I will try to get this on my schedule and pull something together. It shouldn't block this first draft of the template though

Comment (zsd4yr) [reply .github/PULL_REQUEST_TEMPLATE.md]: @RussKie please create an issue for the followup, and then I am satisfied to take this change :)

**Pull request merged**